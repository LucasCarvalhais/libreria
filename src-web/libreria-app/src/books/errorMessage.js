import React from 'react';
export const ErrorMessage = ({ error }) => <div className="message">
    <p>Ocurri√≥ un problema :(</p>
    <p>{error.toString()}</p>
</div>;
