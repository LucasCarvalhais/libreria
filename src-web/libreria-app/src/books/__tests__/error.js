import React from 'react';
import renderer from 'react-test-renderer';
import { ErrorMessage } from '../errorMessage';

const error = {
    error: 'No Network Available',
};

test('errorMessage has a valid snapshot', () => {
    const component = renderer.create(<ErrorMessage {...error} />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});
