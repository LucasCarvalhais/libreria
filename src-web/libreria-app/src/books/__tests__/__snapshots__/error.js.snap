// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`errorMessage has a valid snapshot 1`] = `
<div
  className="message"
>
  <p>
    Ocurri√≥ un problema :(
  </p>
  <p>
    No Network Available
  </p>
</div>
`;
